processor/src/main/java/org/mapstruct/ap/internal/model/source/SourceReference.java

```diff
@@ -108,8 +108,14 @@ public SourceReference build() {
             boolean isValid = true;
             boolean foundEntryMatch;
 
+            String sourceNameTrimmed = sourceName.trim();
+            if ( !sourceName.equals( sourceNameTrimmed ) ) {
+                messager.printMessage( method.getExecutable(), mapping.getMirror(), mapping.getSourceAnnotationValue(),
```

Comment:
Can you format this and the one in `TargetReference` to:

```
messager.printMessage( 
    method.getExecutable(), 
    mapping.getMirror(), 
    mapping.getSourceAnnotationValue(),
    Message.PROPERTYMAPPING_WHITESPACE_TRIMMED, 
    mapping.getSourceName()
);
```

That would be inline with our code style. However, I can also do this during the merge ;)